task/buildah/0.2/buildah.yaml

```diff
@@ -5,6 +5,7 @@ metadata:
     app.kubernetes.io/version: "0.2.1"
     build.appstudio.redhat.com/build_type: "docker"
   annotations:
+    build.appstudio.redhat.com/expires-on: "2025-03-01T00:00:00Z"
```

Comment:
Should we also define `expiry-message` as in the following annotation? https://github.com/konflux-ci/build-definitions/blob/54c1b06f77a1c8eba1baae789a810f60cd10d343/task/buildah-10gb/0.2/buildah-10gb.yaml#L5-L8

Reply:
The default message is ["Upgrade to a newer version of the Task."](https://github.com/enterprise-contract/ec-policies/blob/ce1475d3ab81a7ace697aee65e1e3bd0af895968/policy/release/tasks/tasks.rego#L265), which should work well enough here